### PR TITLE
Fix - Added logout icon in cart and wishlist page and code cleanup

### PR DESCRIPTION
**Pull request by SRUJANA-PENUGONDA13**
## Change log

1. Added log out icon in wishlist and cart page
2. Code clean up

cart
wishlist

### CODE REVIEW

File: css/components/home.css [hunk 1]
```diff
@@ -78,15 +92,19 @@
   padding: 10px;
   border-radius: 10px;
 }
+
 .shp-now-btn a {
   color: var(--text--primary--color);
 }
+
 .title {
   font-size: larger;
```
Comment (viru2001) [inline css/components/home.css]: you should use relative units for font size

File: css/components/product-listing.css [hunk 2]
```diff
@@ -61,6 +64,7 @@
   border: none;
   right: 10px;
 }
+
 .heart-btn .fa-heart {
   color: red;
```
Comment (viru2001) [inline css/components/product-listing.css]: you should use hex or rgb() values for setting color

**Review by viru2001 — COMMENTED** [review]
suggested minor changes. Everything else looks good

File: css/components/home.css (hunk 1 above)
Comment (SRUJANA-PENUGONDA13) [reply css/components/home.css]: Updated

File: css/components/product-listing.css (hunk 2 above)
Comment (SRUJANA-PENUGONDA13) [reply css/components/product-listing.css]: Updated

Comment (SHIKHASINGH1506) [reply css/components/product-listing.css]: After updated code works good

**Comment by SRUJANA-PENUGONDA13** [comment]
Thank You @SHIKHASINGH1506 @viru2001 for reviewing PR

**Pull request merged**